openquake/hazardlib/shakemap/parsers.py

```diff
@@ -144,7 +147,16 @@ def get_array_shapefile(kind, fname):
     """
     fname = path2url(fname)
     if fname.endswith('.zip'):
-        fname = 'zip://' + fname[5:]
+        if sys.platform == 'win32':
+            # fiona cannot automatically unzip, so unzip manually
+            targetdir = tempfile.mkdtemp(
+                dir=config.directory.custom_tmp or None)
```

Comment:
directory.custom_tmp is defined on folder writable from windows user?

Reply:
It is not defined, meaning that the Windows temporary directory is used